_includes/social.html

```diff
@@ -19,12 +19,17 @@
   <meta property="og:url" content="{{ site.url }}{{ page.url }}">
   <meta property="og:title" content="{{ page.title }}">
   <meta property="og:description" content="{{ page.description }}">
+  <meta property="og:type" content="website">
 {% else %}
   <meta property="og:url" content="{{ site.url }}">
   <meta property="og:title" content="{{ site.title }}">
   <meta property="og:description" content="{{ site.description }}">
 {% endif %}
-<meta property="og:image" content="{{ site.url }}{{ site.social_image_path }}">
+<meta property="og:image" content="{{ site.url | replace: 'https://', 'http://' }}{{ site.social_image_path }}">
```

Comment:
Nothing on that page says that the `og:image` property can't be an https URL or that it has to be different than `og:image:secure_url`